### PR TITLE
Clarify why some module-level objects in `typing` have default values

### PR DESCRIPTION
**Pull request by AlexWaygood**
One possible solution to the issue raised in https://github.com/python/typeshed/pull/7036#discussion_r791847978.

cc. @JelleZijlstra

### CODE REVIEW

File: stdlib/@python2/typing_extensions.pyi [hunk 1]
```diff
@@ -33,7 +34,7 @@ def runtime_checkable(cls: _TC) -> _TC: ...
 
 # This alias for above is kept here for backwards compatibility.
 runtime = runtime_checkable
-Protocol: _SpecialForm = ...
+Protocol: _SpecialForm
```
Comment (hauntsaninja) [inline stdlib/@python2/typing_extensions.pyi:37]: Can we re-export Protocol from typing?

**Comment by AlexWaygood** [comment]
I hate it when flake8 complains at me with error codes I introduced.

File: stdlib/@python2/typing_extensions.pyi (hunk 1 above)
Comment (AlexWaygood) [reply stdlib/@python2/typing_extensions.pyi]: It's not over till mypy_primer sings, but looks like we can!

**Comment by AlexWaygood** [comment]
I take it by your silence @JelleZijlstra that you're not a fan of this PR ;) shall I close it?

**Comment by JelleZijlstra** [comment]
Sorry, too many things to look at. I'll review it later today.

**Comment by AlexWaygood** [comment]
> Sorry, too many things to look at. I'll review it later today.

Of course, sorry for nagging! No rush.

**Pull request merged**